views/library.jade

```diff
@@ -2,7 +2,7 @@ extends layout
 
 block content
     .container-fluid
-        h1 My Library
+        h1 Favorites
```

Comment:
Good. Follow up this PR with one changing the file names and references in documentation.